### DISCUSSION TITLE
Problems to read with multiple threads (suscriber for each other)

### DISCUSSION THREAD

**Discussion by AlbertoElDevaluador** [jmmorato/openddsharp#201] [Q&A]
I have problems launching two threads with the connection logic of two DDS subscribers.  The error happens when I post the topic of one of them, even though they are in different domains, it is mysteriously picked up by the one that is not supposed to do it. Do you know why this can happen?

**Comment by mitza-oci** [comment]
How are the OpenDDS transports configured for this scenario?
See https://opendds.readthedocs.io/en/latest-release/devguide/run_time_configuration.html#rtps-udp-transport-configuration-options under "Some implementation notes related to using the rtps_udp transport protocol are as follows..." (if you are using the rtps_udp transport type)

**Reply by AlbertoElDevaluador** [reply]
So... How can I create more than one participant? Should I create more profiles of participants???

**Comment by jmmorato** [comment]
Can you share your code or a reproducer of the issue?
